src/SSHDebugPS/AD7UnixAsyncCommand.cs

```diff
@@ -143,9 +143,25 @@ private void Close()
                     return;
 
                 _isClosed = true;
-            }
 
-            _streamingShell.Close();
+                try
+                {
+                    if (_streamingShell != null && _streamingShell.IsOpen)
```

Comment:
Primitives return values gets converted into nullable when using ?.  
Keeping it as is.